src/components/panel/_macro.njk

```diff
@@ -30,6 +30,8 @@
 
             {% if params is defined and params and params.type == "warn" or params.type == "warn-branded" %}
                 <span class="panel__icon" aria-hidden="true">!</span>
+                <strong>
+                <span class="u-vh">Warning</span>
```

Comment:
I think this should be a param with a default.

```suggestion
                <span class="u-vh">{{ params.iconText | default("Warning: ") }}</span>
```

Reply:
I can sort that out today @jrbarnes9